critiquebrainz/frontend/views/test/test_rate.py

```diff
@@ -47,8 +44,10 @@ def setUp(self):
             full_name="Test License.",
         )
 
-    def test_rate(self):
+    @mock.patch('critiquebrainz.frontend.external.musicbrainz_db.release_group.get_release_group_by_id')
+    def test_rate(self, get_release_group):
         self.temporary_login(self.reviewer)
+        get_release_group.return_value = self.test_entity
```

Comment:
would be nice to `assert_called_with` this mock, and other mocks in similar tests

Reply:
Can you explain what you mean by similar tests ?

Reply:
Also, this was mock turned out to be unneeded so I removed it.